<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tclConsoleDialog</class>
 <widget class="QDialog" name="tclConsoleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>452</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TCL Interpreter console</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="inputLevelLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="commandInputEdit"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="commandButtonLayout">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="evalButton">
         <property name="text">
          <string>Evaluate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="outputLabel">
     <property name="text">
      <string>Interpreter output</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="listView"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="inputLabel">
     <property name="text">
      <string>Command input</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>tclConsoleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>807</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
